mkdocs.yml

```diff
@@ -64,15 +64,17 @@ nav:
       - 'NCCL': software/communication/nccl.md
       - 'RCCL': software/communication/rccl.md
       - 'libfabric': software/communication/libfabric.md
-    - 'Tools':
-      - software/tools/index.md
-      - 'Linaro Forge': software/tools/linaro.md
     - 'Building and Installing software':
       - build-install/index.md
       - 'uenv': build-install/uenv.md
       - 'Containers': build-install/containers.md
       - 'Cray Programming Environment (CPE)': build-install/cpe.md
       - 'Python with pip': build-install/pip.md
+    - 'Debugging and Performance Analysis tools':
+      - software/devtools/index.md
+      - 'Debugging': software/devtools/debug/index.md
+        # - 'Linaro Forge': software/devtools/linaro.md
+      - 'Performance Analysis': software/devtools/perf/index.md
```

Comment:
Would it make sense to merge the contents of `software/devtools/debug/index.md` and `software/devtools/perf/index.md` into `software/devtools/index.md`, and add `perf/linaro.md` etc. as subpages explicitly here so that they show up in the TOC?

Reply:
JG, you were complaining that the table of contents was collapsing when you clicked on links - this is the reason.
I have refactored the docs so that everything shows up in the ToC.